SConstruct

```diff
@@ -2090,6 +2090,10 @@ def removeDirectories(target, source, env):
 
 uninstall = env.Command("uninstall", None, Delete(allfiles))
 env.AddPostAction(uninstall, Action(removeDirectories))
+if env["python_package"] == "full":
+    env.AddPostAction(uninstall, Action("$python_cmd_esc -m pip uninstall Cantera"))
+elif env["python_package"] == "minimal":
+    env.AddPostAction(uninstall, Action("$python_cmd_esc -m pip uninstall Cantera_minimal"))
```

Comment:
I'm not convinced it's worth trying to manage an uninstall from SCons, YMMV. I didn't try either of these commands.